source/usage-examples/deleteMany.txt

```diff
@@ -0,0 +1,72 @@
+=====================
+Delete Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can delete several documents in a collection at once with
+``collection.deleteMany()``.
+The ``deleteMany()`` method uses a query document that you provide
+to match only the subset of the documents in the collection that match
+the query. If you don't provide a query document (or if you provide an
+empty document), MongoDB matches all documents in the collection. All
+matched documents are deleted.
+
+You can also define additional query options such as
+`write timeout <https://docs.mongodb.com/manual/reference/write-concern/#wtimeout>`_
+and
+`write concern <https://docs.mongodb.com/manual/reference/write-concern/#w-option>`_
+to configure the returned document. You should specify these options
+through the ``options`` object passed as the second parameter of the
+``deleteMany()`` method. For detailed reference documentation, see
+`collection.deleteMany() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#deleteMany>`_.
+You can also pass a callback method as an optional third parameter.
+
+``deleteMany()`` can behave in two different ways depending on
+whether or not a callback method is provided:
+
+- if no callback method is provided, ``deleteMany()`` returns a
+  `Promise <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>`_
+  that resolves to an
+  `Object <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+
+- if a callback method is provided, ``deleteMany()`` returns
+  nothing, and instead passes the result object or error object to the
+  provided callback method
+
+The result object contains multiple keys in the event of a successful
```

Comment:
Let's only focus on the interesting bits. Probably deletedCount, maybe there is something in result?